[2022-08-11 19:16:46,392] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_V2.Ingest_from_2016 manual__2022-08-11T00:39:54.349266+00:00 [queued]>
[2022-08-11 19:16:46,408] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_V2.Ingest_from_2016 manual__2022-08-11T00:39:54.349266+00:00 [queued]>
[2022-08-11 19:16:46,409] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 19:16:46,410] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-11 19:16:46,411] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 19:16:46,465] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): Ingest_from_2016> on 2022-08-11 00:39:54.349266+00:00
[2022-08-11 19:16:46,511] {standard_task_runner.py:52} INFO - Started process 1316 to run task
[2022-08-11 19:16:46,560] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_DAG_V2', 'Ingest_from_2016', 'manual__2022-08-11T00:39:54.349266+00:00', '--job-id', '415', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpwl70pgyb', '--error-file', '/tmp/tmp0bm3elen']
[2022-08-11 19:16:46,566] {standard_task_runner.py:80} INFO - Job 415: Subtask Ingest_from_2016
[2022-08-11 19:16:46,789] {task_command.py:371} INFO - Running <TaskInstance: ETL_DAG_V2.Ingest_from_2016 manual__2022-08-11T00:39:54.349266+00:00 [running]> on host ce1be8c59d56
[2022-08-11 19:16:47,064] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=narapady
AIRFLOW_CTX_DAG_ID=ETL_DAG_V2
AIRFLOW_CTX_TASK_ID=Ingest_from_2016
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T00:39:54.349266+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-11T00:39:54.349266+00:00
[2022-08-11 19:17:01,644] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table1_2016.xlsx to s3-bucket-raw-usda
[2022-08-11 19:17:02,964] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table2_2016.xlsx to s3-bucket-raw-usda
[2022-08-11 19:17:04,299] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table3_2016.xlsx to s3-bucket-raw-usda
[2022-08-11 19:17:05,775] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table4_2016.xlsx to s3-bucket-raw-usda
[2022-08-11 19:17:07,163] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table5_2016.xlsx to s3-bucket-raw-usda
[2022-08-11 19:17:08,572] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table6_2016.xlsx to s3-bucket-raw-usda
[2022-08-11 19:17:09,888] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table7_2016.xlsx to s3-bucket-raw-usda
[2022-08-11 19:17:11,251] {logging_mixin.py:115} INFO - Successfully uploaded 2016/Table8_2016.xlsx to s3-bucket-raw-usda
[2022-08-11 19:17:11,260] {python.py:173} INFO - Done. Returned value was: None
[2022-08-11 19:17:11,296] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_DAG_V2, task_id=Ingest_from_2016, execution_date=20220811T003954, start_date=20220811T191646, end_date=20220811T191711
[2022-08-11 19:17:11,347] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-11 19:17:11,398] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
